modules/client/src/client-common/context.ts

```diff
@@ -1,4 +1,4 @@
-import { ContextParams, ContextState } from "./interfaces/context";
+import { ContextParams, ContextState } from "./types/context";
```

Comment:
don't we have any interface left on the clients side?

Reply:
yes, we have some, but are in the interfaces.ts file